docs/site-example/site.conf

```diff
@@ -35,6 +35,14 @@
     -- Wireless channel.
     channel = 1,
 
+    -- List of supported wifi rates (optional)
+    -- Example removes 802.11b compatibility for better performance
+    supported_rates = {6000, 9000, 12000, 18000, 24000, 36000, 48000, 54000},
+
+    -- List of basic wifi rates (optional, recommended if supported_rates is set)
```

Comment:
This comment doesn't match the check_site script: the script makes `basic_rate` mandatory if `supported_rates` is set, not just recommended.